packages/api/src/external/commonwell/patient.ts

```diff
@@ -433,51 +433,49 @@ async function validateCWEnabled({
   patient,
   facilityId,
   forceCW,
-  debug,
+  log,
```

Comment:
debug is disabled on cloud envs, so we were not logging whether it was disabled or not